sdk/monitor/query/MIGRATION.md

```diff
@@ -0,0 +1,210 @@
+# Guide to migrate from `azquery` to `azlogs`, `azmetrics`, and `armmonitor`
+
+With the service team's creation of a new data plane metrics endpoint, the Go SDK team decided to reimagine our Monitor story. After careful deliberation, we decided to restructure Monitor Query to more accurately reflect the [REST API][rest_api].
+
+Logs largely remained the same; the code just moved from `azquery` to [`azlogs`][azlogs].
+
+Metrics has a larger split. Users wanting to use the existing ARM APIs should use the [`armmonitor`][armmonitor] module. `armmonitor` contains all the metrics functionality from `azquery` and more.
+
+[`azmetrics`][azmetrics] contains a [new data plane metrics feature][azmetrics_blog] that wasn't previously available in `azquery`. This allows users to query data from one or multiple resource IDs using a data plane endpoint. `azmetrics` will expand in the future when the service adds more features.
+
+### Name changes
+
+| old module   | old method name |new module | new method name | 
```

Comment:
```suggestion
| Old module   | Old method name |New module | New method name | 
```

Reply:
fixed in [b994525](https://github.com/Azure/azure-sdk-for-go/pull/22753/commits/b994525bed1636c83155a3932ce00aeb00e2fc70)